src/screens/explore-screen/components/ColorTile.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { ReactNode } from 'react'
+
+import {
+  Image,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+  ViewStyle
+} from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+import IconAudioRewardsPill from 'app/assets/images/iconAudioRewardsPill.svg'
+import Text from 'app/components/text/Text'
```

Comment:
Very minor but you should be able to import from the index file directly

```suggestion
import Text from 'app/components/text'
```